bugwarrior/services/gitlab.py

```diff
@@ -42,6 +42,7 @@ class GitlabConfig(config.ServiceConfig, prefix='gitlab'):
     use_https: bool = True
     verify_ssl: bool = True
     project_owner_prefix: bool = False
+    query_string: str = ''
```

Comment:
Any reason not to simply call this `query`, as with github?

Reply:
absolutely not. Will change that!

Reply:
Resolved in 24ef4bf